prow/cmd/crier/main.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/time/rate"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/workqueue"
+
+	prowjobclientset "k8s.io/test-infra/prow/client/clientset/versioned"
+	prowjobinformer "k8s.io/test-infra/prow/client/informers/externalversions"
+
+	"k8s.io/test-infra/prow/crier"
+	"k8s.io/test-infra/prow/logrusutil"
+)
+
+const (
+	resync = 0 * time.Second
```

Comment:
Just noticed this, not even sure what `0` will do but this should be on the order of minutes

Reply:
0 means it will not do resync, per https://github.com/kubernetes/test-infra/pull/8991#pullrequestreview-145564742

I can change the unit to minute